ghost/admin/mirage/fixtures/settings.js

```diff
@@ -81,6 +81,8 @@ export default [
     setting('portal', 'portal_button_style', 'icon-and-text'),
     setting('portal', 'portal_button_icon', null),
     setting('portal', 'portal_button_signup_text', 'Subscribe'),
+    setting('portal', 'portal_signup_terms_html', null),
+    setting('portal', 'portal_signup_checkbox_required', 'false'),
```

Comment:
I see we have other booleans as strings - but just wan to double check that's correct and they get parsed?

Reply:
Yeah, doesn't make much sense. I'll try to update them all to booleans and see if the tests still work. Because that would make more sense.

Reply:
✅